starter-code/index.js

```diff
@@ -1,11 +1,47 @@
 class SortedList {
-  constructor() {}
-  add(item) {}
-  get(pos) {}
-  max() {}
-  min() {}
-  average() {}
-  sum() {}
+  constructor() {
+    this.items  = [];
+    this.length = 0;
+  }
+  add(x){
```

Comment:
and in order to pass the  #add(x) and get(i) test, you need to sort the elements. There is a clue in the terminal when you run the test: `should add a second value to SortedList, sorted`. So we just need to do a simple sort and then you'll be able to pass the tests  

```
this.items.sort(function(a, b) {
      return a - b;
    });
```